scrapy/mail.py

```diff
@@ -5,6 +5,8 @@
 """
 import logging
 
+import boto
```

Comment:
this import should be near twisted import; pep8 [suggests](https://www.python.org/dev/peps/pep-0008/?#imports) the following groups in import blocks:

1) Standard library imports (we consider `six` as stdlib imports in Scrapy).
2) Related third party imports.
3) Local application/library specific imports.

Reply:
`boto` is only used with `SESMailSender`, so the user is not obligated to have it installed. I am moving this import inside the `send` method of the mentioned class (this is done the same way in `S3FeedStorage` for example).

Reply:
Right, a good catch.